packages/plugin-dom-helpers/src/DomHelpers.ts

```diff
@@ -5,6 +5,23 @@ import { Attributes } from '../../plugin-xml/src/Attributes';
 import { Layout } from '../../plugin-layout/src/Layout';
 import { DomLayoutEngine } from '../../plugin-dom-layout/src/DomLayoutEngine';
 import { Parser } from '../../plugin-parser/src/Parser';
+import { Format } from '../../core/src/Format';
+import { Modifiers } from '../../core/src/Modifiers';
+import { flat } from '../../utils/src/utils';
+
+function getAllAttributesModifiers(modifiers: Modifiers): Attributes[] {
+    return flat(
+        modifiers
+            .filter(modifier => modifier instanceof Attributes || modifier instanceof Format)
+            .map((modifier): Attributes[] => {
+                if (modifier instanceof Format) {
+                    return getAllAttributesModifiers(modifier.modifiers);
+                } else {
+                    return [modifier as Attributes];
+                }
+            }),
+    );
```

Comment:
Wow, dans quel cas est-ce que tu parviens à avoir plusieurs modifiers `Attributes` sur le même noeud ?

Reply:
Never, why?
I typecast modifier in order to avoid to instanceof.
I could get this:
```
node
- modifier: Attribute
- modifier: Format
  - modifier: Attribute
- modifier: Format
  - modifier: Attribute
```
```

Reply:
Ok I understand, but then I'm surprised you have to do `as Attributes`. Couldn't you type the `modifier` attribute in the `map` as `Attributes | Format` ?